graphql/ui_plugin_build_baron.go

```diff
@@ -52,24 +52,14 @@ func BbFileTicket(context context.Context, taskId string, execution int) (bool,
 	settings := env.Settings()
 	queue := env.RemoteQueue()
 	bbProject, ok := BbGetProject(settings, t.Project)
-	var webHook evergreen.WebHook
-	flags, err := evergreen.GetServiceFlags()
-	if err != nil {
-		return false, errors.Wrap(err, "error retrieving admin settings")
-	}
-	if flags.PluginAdminPageDisabled {
-		projectRef, err := model.FindOneProjectRef(t.Project)
-		if err != nil {
-			return false, errors.Wrap(err, "problem finding project ref")
-		}
-		webHook = projectRef.TaskAnnotationSettings.FileTicketWebHook
-	} else {
-		webHook = bbProject.TaskAnnotationSettings.FileTicketWebHook
-	}
 	if !ok {
 		return taskNotFound, errors.Errorf("error finding build baron plugin for task '%s'", taskId)
 	}
 
+	webHook, ok := IsWebhookConfigured(t.Project)
+	if !ok {
```

Comment:
I'm not sure this should be treated as an error; it isn't in the original code. We just only hit 63 if it has been.